perf_bench_test.go

```diff
@@ -655,30 +664,23 @@ func bindVMsToSocket() error {
 		return err
 	}
 
-	cpus, err := cpuInfo.SocketCPUs(*bindSocket)
-	if err != nil {
-		return err
-	}
-
-	subCPUs, err = getCoresExceptProfiled(cpuInfo, cpus)
-	if err != nil {
-		return err
+	if socket > -1 {
```

Comment:
add a comment, describing what this logic does

Reply:
Done